src/components/common/Forms/RegisterForm/RegisterForm.jsx

```diff
@@ -189,16 +176,28 @@ export const RegisterForm = () => {
                             
 
                             <CheckboxContainer>
-                                <div onClick={handleToggleCheck}>
-                                    {
-                                        isChecked
-                                            ? <Icon name='checked' size={24} />
-                                            : <Icon name='unchecked' size={24} />
-                                    }
-                                </div>
-                                <ConfirmationText>
-                                    By checking this box, you are creating an account and you agree to the <PolicyLink target='_blank' to='/conditions'>Terms & Conditions</PolicyLink> and <PolicyLink target='_blank' to='/policy'>Privacy Policy</PolicyLink>.
-                                </ConfirmationText>
+                                <label>
+                                    <Checkbox>
+                                        <Field
+                                            name='confirmation'
+                                            type='checkbox'
+                                            onClick={handleToggleCheck}
+                                            value={isChecked}
+                                            style={{display: 'none'}}
+                                        />
+                                        <div>
+                                            {
+                                                isChecked
```

Comment:
`<Icon name={isChecked ? 'checked' : 'unchecked'} size={24} />`

Reply:
Code has been fixed by the comment